cookbook/request-user-for-authentication/README.md

```diff
@@ -0,0 +1,78 @@
+# Requesting a user to authenticate
+
+When your App requires authentication, you should invite the user to login when he tries to access your app features.
+The SignIn App provides a signin form ready to be invoked from other Starcounter apps thanks to the blending approche.
+Here is the pattern for requesting authentication from your app:
+
+When your app, decides that the view cannot be presented without authentication, it redirects to its own "Unauthenticated" view. 
+for exemple: /your-app/partial/unauthenticated?return_uri={?}
+
+<div class="code-name">MainHandlers.cs</div>
+
+```cs
+{
+	if (!IsSignedIn()) {
+		return Self.GET("/your-app/partial/Unauthenticated?return_uri=" + getURL);
+	}
+}
+```
+
+the `getURL` is the Url requested in the original Get request.
+
+
+And then define a new handler for the "Unauthenticated" partial
```

Comment:
And then define -> Then, define
"Unauthenticated" partial -> "Unauthenticated" partial. (period at the end)